staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -84,14 +85,8 @@ func strategicPatchObject(
 	objToUpdate runtime.Object,
 	versionedObj runtime.Object,
 ) (originalObjMap map[string]interface{}, patchMap map[string]interface{}, retErr error) {
-	// TODO: This should be one-step conversion that doesn't require
-	// json marshaling and unmarshaling once #39017 is fixed.
-	data, err := runtime.Encode(codec, originalObject)
-	if err != nil {
-		return nil, nil, err
-	}
 	originalObjMap = make(map[string]interface{})
-	if err := json.Unmarshal(data, &originalObjMap); err != nil {
+	if err := unstructured.DefaultConverter.ToUnstructured(originalObject, &originalObjMap); err != nil {
```

Comment:
I'd like the comment on this method to indicate that this expects a versioned object, not the internal object like it did before.  Also, no other callers in this package?